src/coffee/filter-bar/filter-bar-item.styl

```diff
@@ -1,194 +1,6 @@
-
-  .divider
-    height: 1px
-    margin: 9px 0
-    overflow: hidden
-    background-color: #e5e5e5
-
-  > li > a
-    display: block
-    padding: 3px 20px
-    clear: both
-    font-weight: normal
-    text-decoration: none
-    line-height: 1.42857143
-    white-space: nowrap
-
-  & > li > a
-      transition: all 0.2s
-      color: $DropDownText
-      opacity 1
-
-      &.active
-          color: $DropDownTextHover
-          background-color: $DropDownBgHover
-          background-image: none
-          border-left: 4px solid $FilterBarText
-
-      &:focus,
-      &:hover
-          color: $DropDownTextHover
-          background-color: $DropDownBgHover
-          background-image: none
-          border-left: 4px solid $FilterBarActive
-
-.open
-  > .dropdown-menu
-    display: block
-  > a
-    outline: 0
-
-.dropdown-menu-right
-  left: auto
-  right: 0
-
-.dropdown-menu-left
-  left: 0
-  right: auto
-
-.dropdown-header
-  display: block
-  padding: 3px 20px
-  font-size: 12px
-  line-height: 1.42857143
-  color: #777
-  white-space: nowrap
-
-.dropdown-backdrop
-  position: fixed
-  left: 0
-  right: 0
-  bottom: 0
-  top: 0
-  z-index: 999
-
-.pull-right > .dropdown-menu
-  right: 0
-  left: auto
-
-.dropup,
-.navbar-fixed-bottom .dropdown
-
-  .caret
-    border-top: 0
-    border-bottom: 4px solid
-    content: ""
-
-  .dropdown-menu
-    top: auto
-    bottom: 100%
-    margin-bottom: 2px
-
-@media (min-width: 768px)
-  .navbar-right
-    .dropdown-menu
-      right: 0
-      left: auto
-
-    .dropdown-menu-left
-      left: 0
-      right: auto
-      
-.dropdown-menu
-    background-color: $DropDownBg
-    opacity $DropDownOpacity
-
-    & > li > a
-        transition: all 0.2s
-        color: $DropDownText
-        opacity 1
-
-        &.active
-            color: $DropDownTextHover
-            background-color: $DropDownBgHover
-            background-image: none
-            border-left: 4px solid $FilterBarText
-
-        &:focus,
-        &:hover
-            color: $DropDownTextHover
-            background-color: $DropDownBgHover
-            background-image: none
-            border-left: 4px solid $FilterBarActive
-
-.playerchoice
-    padding-left 5px
-    margin-left 8px
-    margin-right -5px
-    border-left 1px solid rgba(255,255,255,.2)
-
-    img
-        width 20px
-        height 20px
-        margin 7.5px 2px
-        float left
-
-    &:after
-        content ''
-        width 30px
-        height 100%
-        top 0
-        right 0
-        position absolute
-
-.playerchoicemenu
-    font-size 1em
-    text-align left
-
-    & > li > a
-        padding-left 10px
-        padding-right 25px
-        text-overflow ellipsis
-        overflow hidden
-        max-width 200px
-
-        img
-            height 20px
-            width 20px
-            position absolute
-            right 3px
-            -webkit-filter: $PngLogo
\ No newline at end of file
+    &:hover
+        background-color #2B3C48
```

Comment:
I would use an opacity level here to keep the possibility of light and dark themes